### ISSUE TITLE
Cuckoo with 2 VMs but one analyse

### ISSUE THREAD

**Issue by jmg49** [cuckoosandbox/cuckoo#2059]
hi , i' m trying to submit in web mode a sample to two VMs :  1 seven , 1 Xp

in my virtualbox.conf i've set up the 2 vm but when submitting only the first is power up / power down 

analyse cuckoo.log : 

2018-01-15 11:33:25,440 [cuckoo.core.scheduler] INFO: Task #5: acquired machine cuckoo1 (label=seven1)
2018-01-15 11:33:25,450 [cuckoo.auxiliary.sniffer] INFO: Started sniffer with PID 3176 (interface=vboxnet0, host=10.48.54.113)
2018-01-15 11:33:25,451 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2018-01-15 11:33:25,540 [cuckoo.machinery.virtualbox] DEBUG: Starting vm seven1
2018-01-15 11:33:25,828 [cuckoo.machinery.virtualbox] DEBUG: Restoring virtual machine seven1 to sn2
2018-01-15 11:33:53,301 [cuckoo.core.guest] INFO: Starting analysis on guest (id=cuckoo1, ip=10.48.54.113)
2018-01-15 11:33:54,315 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:33:55,330 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:33:56,336 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:33:56,389 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:33:58,394 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:33:59,400 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:00,405 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:01,409 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:02,414 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:03,419 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:04,424 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:05,432 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:06,436 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:07,441 [cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2018-01-15 11:34:12,491 [cuckoo.core.guest] INFO: Guest is running Cuckoo Agent 0.8 (id=cuckoo1, ip=10.48.54.113)
2018-01-15 11:34:22,167 [cuckoo.core.guest] DEBUG: Uploading analyzer to guest (id=cuckoo1, ip=10.48.54.113, monitor=latest, size=3840773)
2018-01-15 11:34:48,671 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:34:48,975 [cuckoo.core.resultserver] DEBUG: LogHandler for live analysis.log initialized.
2018-01-15 11:34:54,490 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:34:57,847 [cuckoo.core.resultserver] DEBUG: New process (pid=848, ppid=4092, name=winhlp32.exe)
2018-01-15 11:34:58,208 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0001.jpg
2018-01-15 11:34:58,213 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 43393
2018-01-15 11:35:00,330 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:06,148 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:12,006 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:17,864 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:23,871 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:35:23,875 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:29,884 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:35:29,887 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:35,891 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:35:35,895 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:40,259 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0002.jpg
2018-01-15 11:35:40,269 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 43818
2018-01-15 11:35:41,392 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0003.jpg
2018-01-15 11:35:41,402 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 43389
2018-01-15 11:35:41,903 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:35:41,906 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:44,869 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0004.jpg
2018-01-15 11:35:44,880 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 43840
2018-01-15 11:35:46,118 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0005.jpg
2018-01-15 11:35:46,130 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 43389
2018-01-15 11:35:47,915 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:35:47,918 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:53,926 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:35:53,929 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:35:59,934 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:35:59,938 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:05,752 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:11,571 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:17,410 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:23,247 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:29,086 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:34,905 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:40,915 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:36:40,918 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:41,661 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0006.jpg
2018-01-15 11:36:41,672 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 43819
2018-01-15 11:36:42,857 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0007.jpg
2018-01-15 11:36:42,865 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 43396
2018-01-15 11:36:46,925 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:36:46,928 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:52,937 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:36:52,940 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:36:58,947 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:36:58,950 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:04,959 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:04,962 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:10,970 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:10,973 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:16,977 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:16,980 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:22,987 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:22,991 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:28,998 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:29,001 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:35,009 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:35,013 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:41,020 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:41,024 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:47,032 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:47,036 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:53,044 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:53,048 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:37:59,054 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:37:59,059 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:05,064 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:05,068 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:11,076 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:11,080 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:17,089 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:17,092 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:23,101 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:23,105 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:29,111 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:29,114 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:35,119 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:35,122 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:41,129 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:41,132 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:47,141 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:47,144 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:53,153 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:53,156 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:38:59,164 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:38:59,168 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:05,176 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:05,180 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:11,189 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:11,192 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:17,201 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:17,204 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:23,213 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:23,218 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:29,225 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:29,228 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:35,238 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:35,241 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:41,249 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:41,254 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:47,260 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:47,264 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:53,276 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:53,280 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:39:59,285 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:39:59,290 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:05,297 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:05,300 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:11,309 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:11,313 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:17,318 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:17,321 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:23,328 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:23,333 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:29,342 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:29,345 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:32,603 [cuckoo.core.resultserver] DEBUG: File upload request for memory/848-1.dmp
2018-01-15 11:40:33,897 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 38406384
2018-01-15 11:40:35,352 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:35,355 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:41,363 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:41,367 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:47,375 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:47,379 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:53,386 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.48.54.113', port=8000): Read timed out. (read timeout=5)",),))
2018-01-15 11:40:53,389 [cuckoo.core.guest] DEBUG: cuckoo1: analysis still processing
2018-01-15 11:40:54,390 [cuckoo.core.guest] INFO: cuckoo1: end of analysis reached!
2018-01-15 11:40:54,409 [cuckoo.core.plugins] ERROR: Unable to stop auxiliary module: Sniffer
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/cuckoo/core/plugins.py", line 162, in stop
    module.stop()
  File "/usr/lib/python2.7/site-packages/cuckoo/auxiliary/sniffer.py", line 154, in stop
    (out, err, faq("permission-denied-for-tcpdump"))
CuckooOperationalError: Error running tcpdump to sniff the network traffic during the analysis; stdout = '' and stderr = 'tcpdump: vboxnet0: That device is not up\n'. Did you enable the extra capabilities to allow running tcpdump as non-root user and disable AppArmor properly (the latter only applies to Ubuntu-based distributions with AppArmor, see also https://cuckoo.sh/docs/faq/index.html#permission-denied-for-tcpdump)?
2018-01-15 11:41:10,587 [cuckoo.machinery.virtualbox] INFO: Successfully generated memory dump for virtual machine with label seven1 to path /root/.cuckoo/storage/analyses/5/memory.dmp
2018-01-15 11:41:10,588 [cuckoo.machinery.virtualbox] DEBUG: Stopping vm seven1
2018-01-15 11:41:17,289 [cuckoo.core.scheduler] DEBUG: Released database task #5
2018-01-15 11:41:17,687 [cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" for task #5
2018-01-15 11:41:17,701 [cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" for task #5
2018-01-15 11:41:17,701 [cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" for task #5
2018-01-15 11:41:17,702 [cuckoo.core.plugins] DEBUG: Executed processing module "DroppedBuffer" for task #5
2018-01-15 11:41:34,741 [cuckoo.processing.memory] DEBUG: Executing volatility 'pslist' module.
2018-01-15 11:41:36,591 [cuckoo.processing.memory] DEBUG: Executing volatility 'psxview' module.
2018-01-15 11:42:01,175 [cuckoo.processing.memory] DEBUG: Executing volatility 'callbacks' module.
2018-01-15 11:42:16,619 [cuckoo.processing.memory] DEBUG: Executing volatility 'ssdt' module.
2018-01-15 11:42:23,700 [cuckoo.processing.memory] DEBUG: Executing volatility 'timers' module.
2018-01-15 11:42:27,309 [cuckoo.processing.memory] DEBUG: Skipping 'messagehooks' volatility module
2018-01-15 11:42:27,309 [cuckoo.processing.memory] DEBUG: Executing volatility 'getsids' module.
2018-01-15 11:42:29,880 [cuckoo.processing.memory] DEBUG: Executing volatility 'privs' module.
2018-01-15 11:42:32,946 [cuckoo.processing.memory] DEBUG: Executing volatility 'malfind' module.
2018-01-15 11:42:40,094 [cuckoo.processing.memory] DEBUG: Skipping 'apihooks' volatility module
2018-01-15 11:42:40,094 [cuckoo.processing.memory] DEBUG: Executing volatility 'dlllist' module.
2018-01-15 11:42:44,238 [cuckoo.processing.memory] DEBUG: Executing volatility 'handles' module.
2018-01-15 11:43:19,434 [cuckoo.processing.memory] DEBUG: Executing volatility 'ldrmodules' module.
2018-01-15 11:43:33,874 [cuckoo.processing.memory] DEBUG: Executing volatility 'mutantscan' module.
2018-01-15 11:43:37,262 [cuckoo.processing.memory] DEBUG: Executing volatility 'devicetree' module.
2018-01-15 11:43:40,822 [cuckoo.processing.memory] DEBUG: Executing volatility 'svcscan' module.
2018-01-15 11:43:43,985 [cuckoo.processing.memory] DEBUG: Executing volatility 'modscan' module.
2018-01-15 11:43:46,798 [cuckoo.processing.memory] DEBUG: Executing volatility 'yarascan' module.
2018-01-15 11:44:14,501 [cuckoo.processing.memory] DEBUG: Executing volatility 'netscan' module.
2018-01-15 11:44:18,745 [cuckoo.core.plugins] DEBUG: Executed processing module "Memory" for task #5
2018-01-15 11:44:19,296 [cuckoo.core.plugins] DEBUG: Executed processing module "MetaInfo" for task #5
2018-01-15 11:44:20,586 [cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" for task #5
2018-01-15 11:44:20,588 [cuckoo.core.plugins] DEBUG: Executed processing module "Procmon" for task #5
2018-01-15 11:44:21,096 [cuckoo.core.plugins] DEBUG: Executed processing module "Screenshots" for task #5
2018-01-15 11:44:21,467 [cuckoo.core.plugins] DEBUG: Executed processing module "Static" for task #5
2018-01-15 11:44:21,517 [cuckoo.core.plugins] DEBUG: Executed processing module "Strings" for task #5
2018-01-15 11:44:21,522 [cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" for task #5
2018-01-15 11:44:21,522 [cuckoo.processing.network] WARNING: The PCAP file does not exist at path "/root/.cuckoo/storage/analyses/5/dump.pcap".
2018-01-15 11:44:21,522 [cuckoo.core.plugins] DEBUG: Executed processing module "NetworkAnalysis" for task #5
2018-01-15 11:44:22,141 [cuckoo.core.plugins] DEBUG: Executed processing module "VirusTotal" for task #5
2018-01-15 11:44:22,142 [cuckoo.core.plugins] DEBUG: Executed processing module "Extracted" for task #5
2018-01-15 11:44:22,143 [cuckoo.core.plugins] DEBUG: Executed processing module "TLSMasterSecrets" for task #5
2018-01-15 11:44:22,247 [cuckoo.core.plugins] DEBUG: Executed processing module "Debug" for task #5
2018-01-15 11:44:22,251 [cuckoo.core.plugins] DEBUG: Running 473 signatures
2018-01-15 11:44:22,735 [cuckoo.core.plugins] DEBUG: Analysis matched signature: antivm_queries_computername
2018-01-15 11:44:22,735 [cuckoo.core.plugins] DEBUG: Analysis matched signature: has_wmi
2018-01-15 11:44:22,736 [cuckoo.core.plugins] DEBUG: Analysis matched signature: antivirus_virustotal
2018-01-15 11:44:22,736 [cuckoo.core.plugins] DEBUG: Analysis matched signature: peid_packer
2018-01-15 11:44:22,737 [cuckoo.core.plugins] DEBUG: Analysis matched signature: memdump_ip_urls
2018-01-15 11:44:22,737 [cuckoo.core.plugins] DEBUG: Analysis matched signature: memdump_urls
2018-01-15 11:44:22,737 [cuckoo.core.plugins] DEBUG: Analysis matched signature: raises_exception
2018-01-15 11:44:22,738 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_handles_1
2018-01-15 11:44:22,738 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_ldrmodules_1
2018-01-15 11:44:22,739 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_malfind_2
2018-01-15 11:44:22,739 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_svcscan_1
2018-01-15 11:44:22,739 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_svcscan_3
2018-01-15 11:44:23,610 [cuckoo.core.plugins] DEBUG: Executed reporting module "JsonDump"
2018-01-15 11:44:24,832 [cuckoo.core.plugins] DEBUG: Executed reporting module "MongoDB"
2018-01-15 11:44:24,833 [cuckoo.core.scheduler] INFO: Task #5: reports generation completed
2018-01-15 11:44:24,852 [cuckoo.core.scheduler] INFO: Task #5: analysis procedure completed

cuckoo.conf  : 

[cuckoo]
# Enable or disable startup version check. When enabled, Cuckoo will connect
# to a remote location to verify whether the running version is the latest
# one available.
version_check = yes

# If turned on, Cuckoo will delete the original file after its analysis
# has been completed.
delete_original = no

# If turned on, Cuckoo will delete the copy of the original file in the
# local binaries repository after the analysis has finished. (On *nix this
# will also invalidate the file called "binary" in each analysis directory,
# as this is a symlink.)
delete_bin_copy = no

# Specify the name of the machinery module to use, this module will
# define the interaction between Cuckoo and your virtualization software
# of choice.
machinery = virtualbox

# Enable creation of memory dump of the analysis machine before shutting
# down. Even if turned off, this functionality can also be enabled at
# submission. Currently available for: VirtualBox and libvirt modules (KVM).
memory_dump = yes

# When the timeout of an analysis is hit, the VM is just killed by default.
# For some long-running setups it might be interesting to terminate the
# monitored processes before killing the VM so that connections are closed.
terminate_processes = no

# Enable automatically re-schedule of "broken" tasks each startup.
# Each task found in status "processing" is re-queued for analysis.
reschedule = no

# Enable processing of results within the main cuckoo process.
# This is the default behavior but can be switched off for setups that
# require high stability and process the results in a separate task.
process_results = yes

# Limit the amount of analysis jobs a Cuckoo process goes through.
# This can be used together with a watchdog to mitigate risk of memory leaks.
max_analysis_count = 0

# Limit the number of concurrently executing analysis machines.
# This may be useful on systems with limited resources.
# Set to 0 to disable any limits.
max_machines_count = 0

# Limit the amount of VMs that are allowed to start in parallel. Generally
# speaking starting the VMs is one of the more CPU intensive parts of the
# actual analysis. This option tries to avoid maxing out the CPU completely.
max_vmstartup_count = 10

# Minimum amount of free space (in MB) available before starting a new task.
# This tries to avoid failing an analysis because the reports can't be written
# due out-of-diskspace errors. Setting this value to 0 disables the check.
# (Note: this feature is currently not supported under Windows.)
freespace = 1024

# Temporary directory containing the files uploaded through Cuckoo interfaces
# (api.py and Django web interface). Defaults to the default temporary
# directory of the operating system (e.g., /tmp on Linux). Overwrite the value
# if you'd like to specify an alternative path.
tmppath =

# Path to the unix socket for running root commands.
rooter = /tmp/cuckoo-rooter

[feedback]
# Cuckoo is capable of sending "developer feedback" to the developers so that
# they can more easily improve the project. This functionality also allows the
# user to quickly request new features, report bugs, and get in touch with
# support in general, etc.
enabled = no
name =
company =
email =

[resultserver]
# The Result Server is used to receive in real time the behavioral logs
# produced by the analyzer.
# Specify the IP address of the host. The analysis machines should be able
# to contact the host through such address, so make sure it's valid.
# NOTE: if you set resultserver IP to 0.0.0.0 you have to set the option
# `resultserver_ip` for all your virtual machines in machinery configuration.
ip = 10.48.54.112

# Specify a port number to bind the result server on.
port = 2042

# Force the port chosen above, don't try another one (we can select another
# port dynamically if we can not bind this one, but that is not an option
# in some setups)
force_port = no

# Maximum size of uploaded files from VM (screenshots, dropped files, log).
# The value is expressed in bytes, by default 128 MB.
upload_max_size = 134217728

[processing]
# Set the maximum size of analyses generated files to process. This is used
# to avoid the processing of big files which may take a lot of processing
# time. The value is expressed in bytes, by default 128 MB.
analysis_size_limit = 134217728

# Enable or disable DNS lookups.
resolve_dns = yes

# Enable PCAP sorting, needed for the connection content view in the web interface.
sort_pcap = yes

[database]
# Specify the database connection string.
# NOTE: If you are using a custom database (different from sqlite), you have to
# use utf-8 encoding when issuing the SQL database creation statement.
# Examples, see documentation for more:
# sqlite:///foo.db
# postgresql://foo:bar@localhost:5432/mydatabase
# mysql://foo:bar@localhost/mydatabase
# If empty, defaults to a SQLite3 database at $CWD/cuckoo.db.
connection =

# Database connection timeout in seconds.
# If empty, default is set to 60 seconds.
timeout = 60

[timeouts]
# Set the default analysis timeout expressed in seconds. This value will be
# used to define after how many seconds the analysis will terminate unless
# otherwise specified at submission.
default = 120

# Set the critical timeout expressed in (relative!) seconds. It will be added
# to the default timeout above and after this timeout is hit
# Cuckoo will consider the analysis failed and it will shutdown the machine
# no matter what. When this happens the analysis results will most likely
# be lost.
critical = 60

# Maximum time to wait for virtual machine status change. For example when
# shutting down a vm. Default is 60 seconds.
vm_state = 60

virtualbox.conf : 

[virtualbox]
# Specify which VirtualBox mode you want to run your machines on.
# Can be "gui" or "headless". Please refer to VirtualBox's official
# documentation to understand the differences.
mode = headless

# Path to the local installation of the VBoxManage utility.
path = /usr/bin/VBoxManage
# If you are running Cuckoo on Mac OS X you have to change the path as follows:
# path = /Applications/VirtualBox.app/Contents/MacOS/VBoxManage

# Default network interface.
interface = vboxnet0

# Specify a comma-separated list of available machines to be used. For each
# specified ID you have to define a dedicated section containing the details
# on the respective machine. (E.g. cuckoo1,cuckoo2,cuckoo3)
machines = cuckoo1,cuckoo2

[cuckoo1]
# Specify the label name of the current machine as specified in your
# VirtualBox configuration.
label = seven1

# Specify the operating system platform used by current machine
# [windows/darwin/linux].
platform = windows

# Specify the IP address of the current virtual machine. Make sure that the
# IP address is valid and that the host machine is able to reach it. If not,
# the analysis will fail.
ip = 10.48.54.113

# (Optional) Specify the snapshot name to use. If you do not specify a snapshot
# name, the VirtualBox MachineManager will use the current snapshot.
# Example (Snapshot1 is the snapshot name):
snapshot = sn2

# (Optional) Specify the name of the network interface that should be used
# when dumping network traffic from this machine with tcpdump. If specified,
# overrides the default interface specified in auxiliary.conf
# Example (vboxnet0 is the interface name):
interface =

# (Optional) Specify the IP of the Result Server, as your virtual machine sees it.
# The Result Server will always bind to the address and port specified in cuckoo.conf,
# however you could set up your virtual network to use NAT/PAT, so you can specify here
# the IP address for the Result Server as your machine sees it. If you don't specify an
# address here, the machine will use the default value from cuckoo.conf.
# NOTE: if you set this option you have to set result server IP to 0.0.0.0 in cuckoo.conf.
# Example:
resultserver_ip =

# (Optional) Specify the port for the Result Server, as your virtual machine sees it.
# The Result Server will always bind to the address and port specified in cuckoo.conf,
# however you could set up your virtual network to use NAT/PAT, so you can specify here
# the port for the Result Server as your machine sees it. If you don't specify a port
# here, the machine will use the default value from cuckoo.conf.
# Example:
resultserver_port =

#(Optional) Set your own tags. These are comma separated and help to identify
# specific VMs. You can run samples on VMs with tag you require.
tags =

# Mostly unused for now. Please don't fill it out.
options =

# (Optional) Specify the OS profile to be used by volatility for this
# virtual machine. This will override the guest_profile variable in
# memory.conf which solves the problem of having multiple types of VMs
# and properly determining which profile to use.
osprofile = Win7SP1x86_23418

[cuckoo2]

label = xp1

platform = windows

ip = 10.48.54.114

snapshot = sn1

interface =

resultserver_ip = 10.48.54.112

resultserver_port =

tags =

options =

osprofile = WinXPSP2x86

[honeyd]
# For more information on this VM please refer to the "services" section of
# the conf/auxiliary.conf configuration file. This machine is a bit special
# in the way that its used as an additional VM for an analysis.
# *NOTE* that if this functionality is used, the VM should be registered in
# the "machines" list in the beginning of this file.
label = honeyd
platform = linux
ip = 192.168.56.102
# The tags should at least contain "service" and the name of this service.
# This way the services auxiliary module knows how to find this particular VM.
tags = service, honeyd
# Not all services actually have a Cuckoo Agent running in the VM, for those
# services one can specify the "noagent" option so Cuckoo will just wait until
# the end of the analysis instead of trying to connect to the non-existing
# Cuckoo Agent. We can't really intercept any inter-VM communication from the
# host / gateway so in order to dump traffic between VMs we have to use a
# different network dumping approach. For this machine we use the "nictrace"
# functionality from VirtualBox (which is basically their internal tcpdump)
# and thus properly dumps inter-VM traffic.
options = nictrace noagent

**Comment by jmg49** [comment]
ok , found a way, 
submitting  2 samples at same times  the two VM start up

**Comment by jbremer** [comment]
Yep, thanks. Closing issue.
